badge_server/badge_server.py

```diff
@@ -67,6 +67,28 @@
 #     'pkg1_api_badge':{},
 # }
 
+REDIS_CACHE = {
+    'opencensus_self_comp_badge': None,
+    'opencensus_google_comp_badge': None,
+}
+
+def fake_redis_get(*args, **kwargs):
+    key = args[2][0]
+    return REDIS_CACHE[key]
+
+
+def fake_redis_set(*args, **kwargs):
+    key = args[2][0]
+    value = str(args[2][1]).encode('utf-8')
+    REDIS_CACHE[key] = value
+
+
+# Patch away the redis connections if run locally
+if os.environ.get('RUN_LOCALLY') is not None:
```

Comment:
Is this only used for test? If it is, why don't we put this in the test cases because it is unrelated to the functionalities of the badge server?

Reply:
Because the `wrapt` magic only works in the same process, and in the test we start a separate process to run the badge server. So if we put that logic in tests, the redis client won't be patched away.